packages/x-tailwindcss/demo/src/components/xds-input-group.vue

```diff
@@ -0,0 +1,122 @@
+<template>
```

Comment:
it is not needed, but it must work if it has, because all the buttons we have in the project have this class, so even we don't include it, it will be there. This example is to show if it works fine even having the `x-button` class